resources/content/schema/base/balancerTargetMetadata.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
this type is for the lb service metadata defined on the target service (when service joins the LB based on selector)